joplin/locations/models.py

```diff
@@ -126,55 +160,29 @@ class LocationPage(JanisBasePage):
     ]
 
 
-class LocationPageRelatedServices(ClusterableModel):
+class LocationPageRelatedServices(Orderable):
 
     page = ParentalKey(LocationPage, related_name='related_services', default=None)
     related_service = models.ForeignKey(
         "base.servicePage",
         on_delete=models.PROTECT,
     )
-    panels = [
-        PageChooserPanel("related_service"),
+    hours_same_as_location = models.BooleanField(default=False, verbose_name="The hours for this service are the same as the location hours")
 
-    ]
+    # slightly clever property + filter for model clean function
```

Comment:
so this is fun: 
a little property that filters fields on the model by their type (@chiaberry I'm sure you can see where else this sort of pattern could be handy, like for Streamfields) 

Then we add a custom clean function for the model (this runs in addition to clean functions specified for forms, check django docs for more details on that lifecycle) and will return an error if the checkbox is not checked and the user also hasn't added any hours for the service.